tools/IoTEdgeDevOps/DevOpsLib/ValidationUtil.cs

```diff
@@ -35,7 +43,7 @@ public static void ThrowIfNull(object value, string field)
         {
             if (value == null)
             {
-                throw new ArgumentNullException("Cannot be null.", field);
+                throw new ArgumentNullException( field, "Cannot be null.");
```

Comment:
``` suggestion
throw new ArgumentNullException( field, "Cannot be null or an empty set.");
```

Reply:
we didn't check if it is empty set; why change the message?